.github/workflows/build.yaml

```diff
@@ -0,0 +1,140 @@
+# https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
+name: Build
+
+on:
+  push:
+    paths-ignore:
+    - '**/*.md'
+  pull_request:
+  workflow_dispatch:
+    # https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        haxe:
+          - 4.1.5
+          - 4.0.5
+          - 3.4.7
+
+    steps:
+    - name: Show environment variables
+      shell: bash
+      run: env | sort
```

Comment:
Not familiar with how GH actions handle secrets, so I have to ask if this risks logging sensitive data?

Reply:
GH masks all secrets by default. You can additionally request masking of non-GH-secrets.

See https://stackoverflow.com/questions/62142092/is-it-okay-to-use-github-secrets-with-a-public-repo and https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#masking-a-value-in-log